### DISCUSSION TITLE
Help wipe encrypt hard disk

### DISCUSSION THREAD

**Discussion by HajjiKakki** [PartialVolume/shredos.x86_64#290] [Q&A]
Hi all, I have a 1TB SATA HDD that is encrypted with Symantec but i forgot the password. Is there a way I can use ShredOS to completely erase all partitions? Because when i tried to wipe i only see the SHREDOS USB DRIVE and other OS drive but i don't see the encrypt one.

**Comment by PartialVolume** [comment]
What's the make and model of the drive?

If it's a self encrypting drive then you may have to reset the drive which will reset the keys which should allow you to access the drive. I don't know whether issuing a secure erase with hdparm has the same effect?

Maybe somebody else that has more experience with self encrypting drives can comment.
